### DISCUSSION TITLE
Edit repository details/Description Feild

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22395] [New to GitHub]
<p>hi everyone.<br>
I would like to insert a link-icon and a link into description field of the repository like below.<br>
<img src="https://user-images.githubusercontent.com/10137/181070743-9c10bc93-b86b-4d54-8359-8dc472c17fe0.png" alt="example" data-base62-sha1="95ohHDCXSE2j9HI3UrvLdkBpjyC" width="210" height="32"><br>
I have tried :link: but it is not the same.<br>
How can I  place an  icon like in the example above with a link?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Use the “Website” field, not the description. :wink:</p>

**Comment by ghost** [comment]
<p>Thanks for your answer.</p>
